src/data/faq_de.json

```diff
@@ -2419,7 +2415,7 @@
                                 "active": false,
                                 "textblock": [
                                     "Ja. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android die Standort-Dienste) eingeschalten haben, werden Begegnungen vom Exposure Notification Framework von iOS/Android aufgezeichnet. Die App selbst muss dafür nicht geöffnet bleiben. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
-                                    "Sicherheitshalber können Sie die App einmal täglich öffnen. Weitere Hinweise finden Sie unter <a href='../#howto' target='_blank' >So funktioniert die App am besten</a>.",
+                                    "Sicherheitshalber können Sie die App einmal täglich öffnen.",

```

Comment:
resolve by changing relative link to point to top level of web, not by deleting the sentence.

Reply:
Corrected as part of commit https://github.com/corona-warn-app/cwa-website/pull/2657/commits/62cef3b52d1590133787805b9c96a624138c2bcc